pkg/plugin/basic/handler.go

```diff
@@ -0,0 +1,148 @@
+package basic
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/hellofresh/janus/pkg/errors"
+	"github.com/hellofresh/janus/pkg/opentracing"
+	"github.com/hellofresh/janus/pkg/render"
+	"github.com/hellofresh/janus/pkg/router"
+	log "github.com/sirupsen/logrus"
+)
+
+// Controller is the api rest controller
+type Controller struct {
+	repo Repository
+}
+
+// NewController creates a new instance of Controller
+func NewController(repo Repository) *Controller {
+	return &Controller{repo}
+}
+
+// Get is the find all handler
+func (c *Controller) Get() http.HandlerFunc {
```

Comment:
Given this is a very basic crud handler, we could borrow some name conventions from some mcv frameworks out there, for this one we could rename to `Index()`